community/contributing/pr_review_guidelines.rst

```diff
@@ -0,0 +1,335 @@
+.. _doc_pr_review_guidelines:
+
+Pull request review guidelines
+==============================
+
+.. note::
+	This page is targeted at engine maintainers responsible for reviewing
+	and approving pull requests. While not all tips and recommendations
+	here are actionable if you are not a maintainer, this can still give
+	you and insight into what goes into successfully merging a PR.
+
+	Even if you are not a maintainer, you can always help by spotting
+	issues in code or problems with the implementation overall, as well
+	as by doing live testing of PRs on your machine and confirming that
+	they work as intended.
+
+If you are a designated Godot contributor and maintainers, you've likely
```

Comment:
Grammar issue, plus I'm not sure if we should list "contributor" here since that word can apply to hundreds of people. Maybe we should also make it clear what maintainers look like on GitHub.

```suggestion
If you are a designated Godot maintainer ("Member" on GitHub), you've likely
```

Reply:
Fair, I'll amend with your suggestion.